src/modules/CRUD/components/Details/Details.js

```diff
@@ -118,23 +117,14 @@ class Details extends React.Component {
 
   renderContent = () => {
     const {
-      match: { params: { action: paramAction, id: paramId } },
+      match: { params: { id } },
       feature,
-      refreshingLayers,
     } = this.props;
 
-    if (paramId === ACTION_CREATE) {
+    if (id === ACTION_CREATE) {
```

Comment:
it's seems not to be an `id`, bad naming ?

Reply:
Actually, the `id` is whatever string as feature ID such as `/monuments/2feac145-2f73-4a3e-9b65-7329944f5e56/`

or the reserved word `create` such as `/monuments/create/`

Reply:
Ok strange behaviour.